### ISSUE TITLE
Modify individual repeating events

### ISSUE THREAD

**Issue by Grant837** [davish/obsidian-full-calendar#308]
Please add the industry standard of allowing the editing of a single instance of a repeating event.  See Google Calendar for a good example.  You can also change all future events, so that past events remain where they where.  Editing included cancelling (deleting) the repeating event.
Also, currently, drag and drop of a repeating event kills the ´repeat´ and you are left with only the moved instance.

**Comment by davish** [comment]
Duplicate of #22

**Comment by Grant837** [comment]
Hi @davish,
I looked at the #22 and that is not the feature I tried unsuccessfully to describe.
Those requests, I believe,  refer to creating a wider variation of repeating events.  
What I ask for was the ability to modify a single one of any already scheduled repeating event to be just one time at a different time/day (or cancel/delete).  
For example, I have a repeating event to go to the gym at 8pm on Wednesday with my son, but sometimes, one week, we have to reschedule. Right now, this does not seem possible.  
Correct me if I am wrong.
Thanks, Grant

**Comment by davish** [comment]
Hi @Grant837,

There are a few parts of your request. The main thrust is that you'd like to make exceptions to recurrence rules and create individual events whose details are copied from the recurring event. This is a feature of RRule, a recurring events specification that is part of the iCalendar spec (ICS). While it may not be reflected in the original description of #22, #22 will be completed using RRule, and so this use-case will be covered.

After #303 is merged, RRule will be implemented on the "back-end", just without any modal support. you'll be able to add `skipDates` to a recurring event to create exceptions.

**Comment by Grant837** [comment]
That´s good news!  Thank you for the clear answer.

**Comment by hycday** [comment]
every issue that talks about an enhanced recurring feature lead to either a solution using an external online calendar... or to this issue's conclusion here.. or to https://github.com/davish/obsidian-full-calendar/pull/303 or to a solution that apparently used to work but not anymore (https://github.com/davish/obsidian-full-calendar/pull/225)

as i'm not sure to understand the state of that one or the https://github.com/davish/obsidian-full-calendar/pull/303, i'm wondering what is the status or latest update please ?
